group.py

```diff
@@ -2,4 +2,14 @@
 
 # Your code to go here...
 
-my_group =
+my_group = [
+    {"Name":"Jill","Age": 26,"Job":"Biologist","Connections":[]},
+    {"Name":"Zalika","Age": 28,"Job":"Artist","Connections":[]},
+    {"Name":"John","Age": 27,"Job":"Writer","Connections":[]},
+    {"Name":"Nash","Age": 34,"Job":"Chef","Connections":[]},
+    ]
+
+my_group[0]["Connections"] = [(my_group[2],"Patner")] # Jill and John
+my_group[0]["Connections"].append((my_group[1],"Friend")) # Jill and John
+
+
```

Comment:
Why did you decide to create a dictionary within a list as opposed to a dictionary within a dictionary? Does this approach have any advantages? Thank you

Reply:
[Suggestion] Perhaps use a list for the "Job" key value of the dictionary so it can be empty (allows for people with no jobs)

Reply:
Why they didn't use classes and it seems it doesn't work.